test/e2e/sourcery.js

```diff
@@ -42,8 +42,27 @@ module.exports = {
             }
         });
         //Assert if expected user is logged in
-        browser.assert.containsText('.user-info__title', 'Demo User')
-            .saveScreenshot(conf.imgpath(browser) + 'Demo.png')
-            .end();
+        browser.assert.containsText('.user-info__title', 'Akvile Jarmalaviciute')
+            .saveScreenshot(conf.imgpath(browser) + 'Demo.png');
+        //Select current date
+        /*
+        browser.element('css selector', '[.calendar__day calendar--today calendar--selected= "7"]', function(result){
+            if(result.status != -1) {
+                browser.click('css selector', '[.calendar__day calendar--today calendar--selected= "7"]');
+            }
+        });   
+        */
+        browser.assert.containsText('.calendar--today', '7');
+        browser.assert.containsText('.main-nav', 'Time Logging');
+        browser.assert.containsText('.main-nav', 'Invoices');
+        browser.assert.containsText('.main-nav', 'Tasks');
+        browser.assert.containsText('.main-nav', 'Projects');
+        browser.assert.containsText('.main-nav', 'Clients');
+        browser.assert.containsText('.main-nav', 'Time Entries');
+        browser.assert.containsText('[href="/time-entries"]', 'Time Entries');
+        browser.assert.containsText('.main-nav__link--active', 'Time Logging');
+        browser.assert.cssProperty('.main-nav__link--active', 'color', 'rgba(64, 76, 237, 1)')
```

Comment:
Not mandatory but in all the lines above you could get rid of "browser" to improve readability.